package.json

```diff
@@ -120,7 +120,7 @@
     "react-addons-shallow-compare": "^15.6.2",
     "react-moment-proptypes": "^1.5.0",
     "react-portal": "^4.1.2",
-    "react-with-styles": "=2.2.0",
+    "react-with-styles": "^3.1.0",
     "react-with-styles-interface-css": "^4.0.0"
```

Comment:
This should be ^3.0.0 to resolve import error

Reply:
So it's not actually this PR that is the issue. It's https://github.com/airbnb/react-dates/pull/1029

Reply:
Okay v16.3.3 is out and should address this.

Reply:
So off of version 16.3.1, i could get it to work by changing the "main" entry in array.prototype.flatten/package.json to "index.js"

it is currently "./"

I haven't seen that anywhere before, not sure if it's a standard thing, or maybe it's my node version (8.9.2), but it seems to be the root cause. I believe you guys switched to using that shim from array_flatten in the react-with-styles-interface-css update.